lib/components/form/user-trip-settings.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react'
+import { Button } from 'react-bootstrap'
+import { connect } from 'react-redux'
+
+import Icon from '../narrative/icon'
+import {
+  clearDefaultSettings,
+  resetForm,
+  storeDefaultSettings
+} from '../../actions/form'
+import { getTripOptionsFromQuery, isNotDefaultQuery } from '../../util/query'
+
+class UserTripSettings extends Component {
```

Comment:
Could you add javadoc to this component?

Reply:
I added the jsdoc for the component.